### PR TITLE
Modify Camera component to upload image as a file object

### PR DESCRIPTION
**Pull request by kart1ka**
I have modified Camera component to upload image as a file object instead of base64 encoded string because file object is smaller in size and easier to handle in the backend as compared to base64 encoded string.

### CODE REVIEW

File: src/form-elements/index.jsx [hunk 1]
```diff
@@ -161,7 +161,7 @@ class PhoneNumber extends React.Component {
 
   render() {
     const props = {};
-    props.type = 'text';
+    props.type = 'number';
```
Comment (Kiho) [inline src/form-elements/index.jsx:164]: I like to keep text here number type is  not allowing formatted text

File: webpack.config.js [hunk 2]
```diff
@@ -58,8 +58,8 @@ module.exports = {
     },
     static: './public',
     open: true,
-    proxy: {
-      "/api/*": "http://127.0.0.1:5005"
-    }
+    // proxy: {
```
Comment (Kiho) [inline webpack.config.js:61]: This should be enabled for posting data when develop

File: src/form-elements/index.jsx [hunk 3]
```diff
@@ -695,7 +683,7 @@ class Camera extends React.Component {
               {this.state.img && (
                 <div>
                   <img
-                    src={this.state.img}
+                    src={URL.createObjectURL(this.state.img)}
```
Comment (Kiho) [inline src/form-elements/index.jsx:686]: Need to release Object URL after image loading

**Comment by Kiho** [comment]
Can you consider to fix those issues I commented?

**Comment by kart1ka** [comment]
Okay. I will fix those issues.

**Comment by kart1ka** [comment]
I have fixed the issues.
Please accept the PR.
Thanks.

**Comment by Kiho** [comment]
Thanks for PR.

**Pull request merged**